pkg/tools/tools.go

```diff
@@ -0,0 +1,8 @@
+// +build tools
```

Comment:
I prefer `dependencymagnet`, it's more self-explanatory IMO

Reply:
definitely, changed :+1: